tinyman/liquid_staking/talgo_client.py

```diff
@@ -0,0 +1,223 @@
+from algosdk import transaction
+from algosdk.encoding import decode_address, encode_address
+from .base_client import BaseClient
+
+
+class TAlgoClient(BaseClient):
+
+    keyreg_lsig = transaction.LogicSigAccount(b"\n\x81\x01C")
```

Comment:
This isn't used anymore actually. The lines with `self.add_key(account_address, self.keyreg_lsig)` aren't needed.